### PR TITLE
Update Templates Readme - Github Repo links

### PR DESCRIPTION
**Pull request by DrW3RK**
When someone downloads the Polkadot SDK repo and navigates to the templates folder, the Readme instructions do not work. There is a getting started script of the Polkadot SDK readme which can be overlooked (and also it covers only minimal template and not the parachain/solochain templates).

The instructions of the Readme files are updated such that they work for anyone on 
https://github.com/paritytech/polkadot-sdk
https://github.com/paritytech/polkadot-sdk-minimal-template
https://github.com/paritytech/polkadot-sdk-parachain-template
https://github.com/paritytech/polkadot-sdk-solochain-template

### CODE REVIEW

**Review by kianenigma — COMMENTED** [review]
Ahh I see what you mean now. These READMEs are meant to be seen from the perspective of the real fork, not here. 

I would say we should generally discourage anyone from cloning the entirety of `polkadot-sdk` to use the templates. 

So, if anything, I would add the opposite in `templates/README.md` that warns users to not use these templates, and instead go to the repos you have linked. 

cc @rzadp

**Comment by rzadp** [comment]
> So, if anything, I would add the opposite in `templates/README.md` that warns users to not use these templates, and instead go to the repos you have linked.

I think it makes sense, especially since if you clone a template repo, you would get the code originated from the stable branch - as opposed to cloning the default branch of `polkadot-sdk` and getting the templates from there.

File: templates/minimal/README.md [hunk 1]
```diff
@@ -45,6 +53,8 @@ packages required to compile this template - please take note of the Rust compil
 cargo build --release
 ```
 
+If you are in `polkadot-sdk` repo, please specify the package name before building. `cargo build --package minimal-template-node --release`
```
Comment (kianenigma) [inline templates/minimal/README.md:56]:
```suggestion
```

File: templates/minimal/README.md [hunk 2]
```diff
@@ -37,6 +37,14 @@ A Polkadot SDK based project such as this one consists of:
 * 🛠️ Depending on your operating system and Rust version, there might be additional
 packages required to compile this template - please take note of the Rust compiler output.
 
+Fetch minimal template code (you can ignore this step if you cloned the entire `polkadot-sdk` repo):
```
Comment (kianenigma) [inline templates/minimal/README.md:40]:
```suggestion
Fetch minimal template code:
```

File: templates/parachain/README.md [hunk 3]
```diff
@@ -39,6 +39,14 @@ A Polkadot SDK based project such as this one consists of:
 * 🛠️ Depending on your operating system and Rust version, there might be additional
 packages required to compile this template - please take note of the Rust compiler output.
 
+Fetch parachain template code (you can ignore this step if you cloned the entire `polkadot-sdk` repo):
```
Comment (kianenigma) [inline templates/parachain/README.md:42]:
```suggestion
Fetch parachain template code:
```

File: templates/parachain/README.md [hunk 4]
```diff
@@ -47,6 +55,8 @@ packages required to compile this template - please take note of the Rust compil
 cargo build --release
 ```
 
+If you are in `polkadot-sdk` repo, please specify the package name before building. `cargo build --package parachain-template-node --release`
```
Comment (kianenigma) [inline templates/parachain/README.md:58]:
```suggestion
```

File: templates/solochain/README.md [hunk 5]
```diff
@@ -23,14 +23,24 @@ packages required to compile this template. Check the
...
 
-Use the following command to build the node without launching it:
+🔨 Use the following command to build the node without launching it:
 
 ```sh
 cargo build --release
 ```
 
+If you are in `polkadot-sdk` repo, please specify the package name before building. `cargo build --package solochain-template-node --release`
+
```
Comment (kianenigma) [inline templates/solochain/README.md:43]:
```suggestion
```

File: templates/parachain/README.md [hunk 6]
```diff
@@ -47,6 +55,8 @@ packages required to compile this template - please take note of the Rust compil
...
+
```
Comment (kianenigma) [inline templates/parachain/README.md:59]:
```suggestion
```

File: templates/minimal/README.md [hunk 7]
```diff
@@ -45,6 +53,8 @@ packages required to compile this template - please take note of the Rust compil
 cargo build --release
 ```
 
+If you are in `polkadot-sdk` repo, please specify the package name before building. `cargo build --package minimal-template-node --release`
+
```
Comment (kianenigma) [inline templates/minimal/README.md:57]:
```suggestion
```

**Review by kianenigma — APPROVED** [review]

**Review by lexnv — APPROVED** [review]

**Comment by DrW3RK** [comment]
Noticed the markdownlint check failed. Fixed it now. Hope this can be merged now. https://github.com/paritytech/polkadot-sdk/actions/runs/10734180797/job/29768873243?pr=5381

**Pull request merged**